tensorflow/lite/micro/tools/make/templates/Makefile.tpl

```diff
@@ -11,6 +26,10 @@ CCFLAGS += %{CC_FLAGS}%
 
 LDFLAGS += %{LINKER_FLAGS}%
 
+
+# library to be generated
+MICROLITE_LIB = libtensorflow-microlite.a
```

Comment:
is this a lib fro the framework or for the runnable binary?

If it's just a lib, it doesn't need to be part of generated project, unless we have a rule for generated lib project (e.g. without example code).

Reply:
hmm, this is a good point. Actually, what I want is to generate the project without example code. I was using this in my own example. I'll update this to build the library by ignoring the example directory. This way the library can be built and linked against for any of the projects.